### PR TITLE
Update cicd for Hosted and non-hosted agent FAQs

### PR DESCRIPTION
**Pull request by v-tanba**

### CODE REVIEW

File: 03-Security-In-CICD/Readme.md [hunk 1]
```diff
@@ -98,8 +99,9 @@ When the pipeline executes, SVTs will scan the specified set of resources.
 Along with input parameter, you can check for below options
 <br/>**Enable OMS Logging:** Switch to enable this task to publish SVT evalution results to an OMS workspace. Steps to configure OMS credential are explained in Step-4
 <br/>**Aggregate Control Status:** Switch to aggregate the SVTs control output. When this is turned off it would show all the failed individual controls in the task summary output.
+<br/>**Do not auto-update AzSDK:** Switch to toggle auto update of AzSDK and required AzureRM modules on the build server. Keep this un-checked for Hosted VS2017 and while using SVT task fot the first time.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: And also when ever there you need to update the version of AzSDK on the Non-Hosted Agents. I think you should append this as well.

File: 03-Security-In-CICD/Readme.md [hunk 2]
```diff
@@ -185,7 +187,7 @@ Linking to the release definition:
 
 **Step-6**: Save the Release Definition.
   
-![03_Save_Release_Defination](../Images/03_Save_Release_Defination.png)  
+![03_Save_Release_Defination](../Images/03_Save_Release_Defination.PNG)
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: as suggested by Manish. Please correct the typo in the file name

File: 03-Security-In-CICD/Readme.md [hunk 3]
```diff
@@ -240,6 +243,21 @@ what we have seen in the case of ad hoc SVT runs:
 
 [Back to top...](Readme.md#contents)
 
+
+### FAQs
+#### I have enabled AzSDK_SVTs task in my release pipeline . I am getting an error ‘The specified module 'AzSDK' was not loaded because no valid module file was found in any module directory’.  How do I resolve this issue?
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: check the space post release pipeline in the first sentence before .

File: 03-Security-In-CICD/Readme.md [hunk 4]
```diff
@@ -240,6 +243,21 @@ what we have seen in the case of ad hoc SVT runs:
...
+- Go to ‘AzSDK_SVTs’ task in your release definition.
+- Make sure that the check box  ‘Do not auto-update AzSDK’ is unchecked.
+ 
+#### I have enabled AzSDK_SVTs task in my release pipeline. It is taking too much time every time I queue a release, how can I reduce that time?
+- Go to ‘AzSDK_SVTs’ task in your release definition.
+- Mark the check box ‘Do not auto-update AzSDK’ as checked. 
+- This will help you save some time by not re-installing the AzSDK from scratch in every run.
+> **Note:** You will need to keep the above checkbox unchecked if you are running the AzSDK_SVTs task on any release agent for the first time OR you are running the task on Hosted VS2017 agent.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: Or you could simply say that for Non-Hosted if you are alraedy running onthe latest modules, then uncheck this. And this needs to be remain unchecked while using Hosted/ Hosted 2017 build agents.

File: 03-Security-In-CICD/Readme.md [hunk 5]
```diff
@@ -240,6 +243,21 @@ what we have seen in the case of ad hoc SVT runs:
...
+
+
+[Back to top...](Readme.md#contents)
+
 # Security Verification Tests (SVTs) in Jenkins pipeline (Preview)
 	
 > Note : AzSDK plugin requires PowerShell to be present on Jenkins Server. Therefore, the plugin is currently supported for Windows machines only.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: Do you want to add a link on how to get AzSDK started on Jenkins?

Comment (v-tanba) [reply 03-Security-In-CICD/Readme.md]: Done with the changes.

File: 03-Security-In-CICD/Readme.md [hunk 6]
```diff
@@ -154,8 +156,7 @@ next to it so that it gets masked.
 (You may skip this step in a first-pass exploration of CICD integration of SVTs.) 
 This feature enables you to set up online policies. 
 This enables the CICD extension to use org-specific policies. 
-<!-- #TODO# Clarify. Need IT-specific instructions too! Or it should just work for IT. --> 
-
+To use org-specific policies, you can get your org-specific url from AzSDKSettings.json file under 'OnlinePolicyStoreUrl' parameter.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: we should provide the location of setting file..how would user know the location of AzSDKSettings.json

File: 03-Security-In-CICD/Readme.md [hunk 7]
```diff
@@ -240,6 +242,21 @@ what we have seen in the case of ad hoc SVT runs:
 
 [Back to top...](Readme.md#contents)
 
+
+### FAQs
+#### I have enabled AzSDK_SVTs task in my release pipeline. I am getting an error ‘The specified module 'AzSDK' was not loaded because no valid module file was found in any module directory’. How do I resolve this issue?
+- Go to ‘AzSDK_SVTs’ task in your release definition.
+- Make sure that the check box  ‘Do not auto-update AzSDK’ is unchecked.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: Give little more background on how this switch would help in fixing the issue. It just keeps the user wondering on how did it fix or going to fix?

File: 03-Security-In-CICD/Readme.md [hunk 8]
```diff
@@ -240,6 +242,21 @@ what we have seen in the case of ad hoc SVT runs:
 
 [Back to top...](Readme.md#contents)
 
+
+### FAQs
+#### I have enabled AzSDK_SVTs task in my release pipeline. I am getting an error ‘The specified module 'AzSDK' was not loaded because no valid module file was found in any module directory’. How do I resolve this issue?
+- Go to ‘AzSDK_SVTs’ task in your release definition.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: There is no task with the name AzSDK_SVTs...Please update with the actual task name. You could actually start with explaining why this problem is happening first

File: 03-Security-In-CICD/Readme.md [hunk 9]
```diff
@@ -240,6 +242,21 @@ what we have seen in the case of ad hoc SVT runs:
...
+- Make sure that the check box  ‘Do not auto-update AzSDK’ is unchecked.
+ 
+#### I have enabled AzSDK_SVTs task in my release pipeline. It is taking too much time every time I queue a release, how can I reduce that time?
+- Go to ‘AzSDK_SVTs’ task in your release definition.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: Same as above

File: 03-Security-In-CICD/Readme.md (hunk 5 above)
Comment (sbyna-msft) [reply 03-Security-In-CICD/Readme.md]: " Therefore, the plugin is currently supported for Windows machines only.	" This statement is not required.

**Pull request merged**